File: Python-appium-test-script/script/common/driver.py
import os

from appium import webdriver

server_url = 'http://127.0.0.1:4723/wd/hub'

config = {
    'platformName': 'Android',
    'deviceName': 'emulator-5554',
    'platformVersion': '8.0.0',
    'app': os.path.abspath('VHomeTV_phone_20200506_v0.5.4.apk'),
    #dev的activity : 'com.tolka.woctv.activity.s'
    'appWaitActivity': 'com.tolka.woctv.activity.t',
    'unicodeKeyboard': 'false',
    'resetKeyboard': 'false',
    'automationName': 'uiautomator2',
    'newCommandTimeout': '60000',
}

def get_driver():
	return webdriver.Remote(server_url, config)

File: Python-appium-test-script/script/Driver.java
import static org.junit.Assert.*;
import org.junit.Before;
import org.junit.Test;
import org.testng.annotations.AfterClass;
import io.appium.java_client.AppiumDriver;
import io.appium.java_client.android.AndroidDriver;
import java.net.URL;
import org.openqa.selenium.remote.CapabilityType;
import org.openqa.selenium.remote.DesiredCapabilities;

public class Driver{

    public AppiumDriver driver;

    @Before
    public void setUp(){
    DesiredCapabilities dc = new DesiredCapabilities();
    dc.setCapability("platformName","Android");
    dc.setCapability("deviceName", "emulator-5554");
    dc.setCapability("platformVersion", "9");
    dc.setCapability("appPackage","tv.tolka.vhometv");
    dc.setCapability("appWaitActivity","com.tolka.woctv.activity.s");
    dc.setCapability("sessionOverride",true);
    driver = new AndroidDriver(new URL("http://127.0.0.1:4723/wd/hub"),dc);




}

    @Test
    public void testplus() {
    
        driver.findElementById("tv.tolka.vhometv:id/btn_one").click();
        
    }
    @AfterClass
    public void tearDown() 
    {
        driver.quit();
    }



}